src/screens/GroupInfo/index.js

```diff
@@ -173,6 +174,10 @@ const GroupInfo = () => {
     handleMember();
   }, []);
 
+  React.useEffect(() => {
+    getMembersList();
+  }, []);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems you have added a new useEffect hook that calls `getMembersList()` without any dependencies. This will cause `getMembersList()` to be called on every render. If this is intended, then it's fine. However, if `getMembersList()` should only be called once when the component mounts, you should add an empty dependency array `[]` to the useEffect hook.
```diff
- React.useEffect(() => {
-   getMembersList();
- }, []);
+ React.useEffect(() => {
+   getMembersList();
+ }, []); // Add dependencies here if any
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->